wizard-js/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "google-labeler-wizard-js",
+  "version": "1.0.0",
+  "description": "Labeler wizard",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
*cough* Can we add tests please.

Reply:
Personally, I was hoping to add tests as a later follow-on. (Adding tests now delays my work on versions of this script for other languages, which in turn delays us bringing proper test-tracking functionality online.) To be clear - I **do** want to add tests at some point, _just not right now_.

That said: if @jmdobry disagrees with that approach, I'll take the time now and add tests.